demo/aws/terraform-sample.tfvars

```diff
@@ -0,0 +1,5 @@
+access_key = "XXX"  # replace with actual value
+secret_key = "XXX"  # replace with actual value
+ami_id = "ami-0ff4c8fb495a5a50d"
```

Comment:
Please add a comment that this IAM is for London region.

Reply:
Added.